plugins/communicators/ssh/communicator.rb

```diff
@@ -348,6 +348,14 @@ def connect(**opts)
         auth_methods << "publickey" if ssh_info[:private_key_path]
         auth_methods << "password" if ssh_info[:password]
 
+        # yanked directly from ruby's Net::SSH, but with `none` last
+        # TODO: Remove this once the next release (> 4.1.0) of Net::SSH is released
+        cipher_array = %w(aes128-cbc 3des-cbc blowfish-cbc cast128-cbc
```

Comment:
We don't want to make a static copy of the cipher list here. Instead, we should directly dup the array from the constant in the net-ssh library and move the `none` value to the end of the new array.